cmd/glasskube/cmd/version.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/glasskube/glasskube/internal/cliutils"
+	"github.com/glasskube/glasskube/internal/config"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var versioncmd = &cobra.Command{
+	Use:    "version",
+	Short:  "Print the version of glasskube and package-operator",
+	Long:   `Print the version of glasskube and package-operator`,
+	PreRun: cliutils.SetupClientContext(true),
+	Run: func(cmd *cobra.Command, args []string) {
+		glasskubeVersion := config.Version
+		fmt.Fprintf(os.Stderr, "glasskube: v%s\n", glasskubeVersion)
+		operatorVersion, err := getPackageOperatorVersion(cmd.Context())
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "✗ no deployments found in the glasskube-system namespace\n")
+		} else {
+			fmt.Fprintf(os.Stderr, "package-operator: %s\n", operatorVersion)
+		}
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(versioncmd)
+}
+
+func getPackageOperatorVersion(ctx context.Context) (string, error) {
+	config, _ := cliutils.RequireConfig("")
```

Comment:
Please use `client.ConfigFromContext(ctx)` to get the config from the context. See github.com/glasskube/glasskube/pkg/client